### PR TITLE
dev-python/watchfiles: Keyword 0.16.1 sparc, #852878

### PR DESCRIPTION
**Pull request by matoro**

### CODE REVIEW

File: dev-util/maturin/maturin-0.13.1.ebuild [hunk 1]
```diff
@@ -343,7 +343,7 @@ LICENSE="
 	BSD CC0-1.0 ISC MIT MPL-2.0 openssl unicode
 	doc? ( CC-BY-4.0 OFL-1.1 )"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~sparc ~x86"
```
Comment (ionenwks) [inline dev-util/maturin/maturin-0.13.1.ebuild:346]: Does this really work? Sparc fixes are in maturin-0.13.2_beta1 (okay to keyword that one instead)

Comment (matoro) [reply dev-util/maturin/maturin-0.13.1.ebuild:346]: Hmm, it seems it did install beta1, but `nattka apply` chose to add the kw to this version instead.  How should I have invoked it instead?  I used `nattka apply -a sparc 852878`.

Comment (ionenwks) [reply dev-util/maturin/maturin-0.13.1.ebuild:346]: (theory fixes anyway, upstream didn't get to test it afaik)

Comment (ionenwks) [reply dev-util/maturin/maturin-0.13.1.ebuild:346]: Ah okay, I believe it just doesn't try to keyword unkeyworded ebuilds given these are often live ebuilds. Probably should just do it manually.

Comment (matoro) [reply dev-util/maturin/maturin-0.13.1.ebuild:346]: It passes tests on real hardware.  Used rust-bin-1.62.1.

Comment (ionenwks) [reply dev-util/maturin/maturin-0.13.1.ebuild:346]: Nice, I'll drop a quick note upstream that it's fine.

**Review by ionenwks — APPROVED** [review]

**Pull request closed without merging**